services/Makefile

```diff
@@ -2,6 +2,9 @@ NODE:=../tools/node
 COMPOSE:=../tools/compose
 IMAGE_NAME:=jenkinsciinfra/evergreen-backend
 DB_DUMP=initdb.d/db.sql
+ifndef SENTRY_URL
+  SENTRY_URL='https://publickey@sentry.io/projectid'
```

Comment:
I think it would be best to incorporate this into `config/default.json` and then allow an environment override for production environments like with `EVERGREEN_JWT_SECRET`